polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -31,48 +29,32 @@
   @media #{$p-breakpoints-sm-up} {
     border-radius: var(--p-border-radius-2);
 
+    @include shadow-bevel(
+      $boxShadow: var(--p-shadow-xs),
+      $borderRadius: var(--p-border-radius-3),
+      $zIndex: 101
+    );
+
     .Section:first-child {
-      border-top-left-radius: var(--p-border-radius-2);
-      border-top-right-radius: var(--p-border-radius-2);
+      border-top-left-radius: var(--p-border-radius-3);
+      border-top-right-radius: var(--p-border-radius-3);
     }
 
     .Section:last-child {
-      border-bottom-left-radius: var(--p-border-radius-2);
-      border-bottom-right-radius: var(--p-border-radius-2);
-    }
-
-    #{$se23} & {
-      @include shadow-bevel(
-        $boxShadow: var(--p-shadow-xs),
-        $borderRadius: var(--p-border-radius-3),
-        $zIndex: 101
-      );
-
-      // stylelint-disable-next-line -- se23 override
-      .Section:first-child {
-        border-top-left-radius: var(--p-border-radius-3);
-        border-top-right-radius: var(--p-border-radius-3);
-      }
-
-      // stylelint-disable-next-line -- se23 override
-      .Section:last-child {
-        border-bottom-left-radius: var(--p-border-radius-3);
-        border-bottom-right-radius: var(--p-border-radius-3);
-      }
+      border-bottom-left-radius: var(--p-border-radius-3);
+      border-bottom-right-radius: var(--p-border-radius-3);
     }
   }
 
   @media print {
     box-shadow: none;
 
-    #{$se23} & {
-      @include shadow-bevel(
-        $boxShadow: none,
-        $borderRadius: var(--p-border-radius-0-experimental),
-        $border: none,
-        $content: none
-      );
-    }
+    @include shadow-bevel(
+      $boxShadow: none,
+      $borderRadius: var(--p-border-radius-0-experimental),
+      $border: none,
+      $content: none
```

Comment:
I think the `$border` and `$content` args are passed into the mixin to ignore the [shadow bevel opt-out](https://github.com/Shopify/polaris/pull/10167/files/2cf64e80253c5a314d58601fd1216fa6803c63b0#r1301993816) and can be removed.